2019/harness/test.js

```diff
@@ -19,6 +19,12 @@
 
 var XHR_TIMEOUT_LIMIT = 5000;
 
+if (!Element.prototype.requestFullscreen) {
+  Element.prototype.requestFullscreen = Element.prototype.mozRequestFullscreen ||
+                                        Element.prototype.webkitRequestFullscreen ||
```

Comment:
Both above shouldn't be needed because support for this API is already around for a while, and I doubt that we want to test old browser versions.